### PR TITLE
Common weights for JS pages

### PR DESCRIPTION
**Pull request by svrnm**
Contributes to #2829

### CODE REVIEW

File: content/en/docs/instrumentation/js/manual.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Instrumentation
-aliases: [/docs/instrumentation/js/api/tracing]
-weight: 4
+title: Manual
+aliases: [/docs/instrumentation/js/api/tracing, /docs/instrumentation/js/instrumentation/]
```
Comment (chalin) [inline content/en/docs/instrumentation/js/manual.md:3]:
```suggestion
aliases:
  - /docs/instrumentation/js/api/tracing
  - /docs/instrumentation/js/instrumentation
```

**Review by chalin — APPROVED** [review]
See inline comment.

**Comment by chalin** [comment]
Oh, for the broken links: you'll need to use the new canonical path to the "Manual" page.

**Comment by svrnm** [comment]
> Oh, for the broken links: you'll need to use the new canonical path to the "Manual" page.

forgot that one, should be fixed now :)

**Review by cartermp — APPROVED** [review]

**Pull request merged**